rosdep/python.yaml

```diff
@@ -4135,6 +4135,18 @@ python-setuptools:
     xenial_python3: [python3-setuptools]
     yakkety: [python-setuptools]
     zesty: [python-setuptools]
+python-sexpdata:
+  debian:
+    '*':
+      pip:
+        packages: [sexpdata]
+    buster: [python-sexpdata]
```

Comment:
I'm sorry to keep iterating here, but I actually think we should reverse this.  That way it will be more future proof.  Something like this instead:

```
    debian:
        '*': [python-sexpdata]
        jessie:
            pip:
                packages: [sexpdata]
       stretch:
           pip:
               packages: [sexpdata]
```